openmm_septop/python_tutorial.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Add details on the analysis of the calculations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>